pkg/sync/sync_context.go

```diff
@@ -743,6 +748,7 @@ func (sc *syncContext) runTasks(tasks syncTasks, dryRun bool) runState {
 	if runState == successful && createTasks.Any(func(t *syncTask) bool { return t.needsDeleting() }) {
 		var wg sync.WaitGroup
 		for _, task := range createTasks.Filter(func(t *syncTask) bool { return t.needsDeleting() }) {
+			task := task
```

Comment:
This line is not needed, same as above.

Reply:
I understand.
Thank you comments.

I think that's been fixed with this commit.
https://github.com/argoproj/gitops-engine/ pull/51/commits/c5d445f642a1283a09aa0c 0056274e1e6fe17997